packages/forms-web-app/src/views/register/behalf/confirmation.njk

```diff
@@ -20,15 +20,23 @@
         We have sent a confirmation email to: {{ email }}
       </p>
 
-      <h2 class="govuk-heading-m">What happens next</h2>
-      <p>You are now registered to comment on the {{ projectName }} project once the Examination stage starts.</p>
-
-      <h2 class="govuk-heading-m">How and when to submit further comments</h2>
-      <p>You must submit your comments by completing the form on the project page. You will need your interested party reference number to make sure your comments are included in the Examination of the application.</p>
-      <p>We will email to let you know when the Examination for this project starts.</p>
-      <p>You can check our guide to <a href="/interested-party-guide">find out more about having your say during the Examination of the application, (opens in new tab)</a>.</p>
-
-      <a href="/overview/{{ caseRef }}" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+     <h2 class="govuk-heading-m">What happens next</h2>
+      <p>You are now registered to have your say. This means you can now:</p>
```

Comment:
could you please add the respective class for paragraph and href "govuk-body" & "govuk-link"